api/coreservice.go

```diff
@@ -268,10 +269,10 @@ func (core *coreService) Account(addr address.Address) (*iotextypes.AccountMeta,
 	if err != nil {
 		return nil, nil, status.Error(codes.NotFound, err.Error())
 	}
+	// TODO: deprecate nonce field in account meta
 	accountMeta := &iotextypes.AccountMeta{
 		Address:      addrStr,
 		Balance:      state.Balance.String(),
-		Nonce:        state.Nonce,
```

Comment:
for safety better do this in another PR as TODO mentioned, my worry is that this might breaking the API behavior, some user may be using `meta.Nonce` from returned value and this will break their code?